Resources/doc/reference/form_types.rst

```diff
@@ -193,7 +193,7 @@ sonata_type_translatable_choice
 
 .. warning::
 
-    ``sonata_type_translatable_choice`` is deprecated and will be removed in 3.0. Use form type ``choice`` with ``translation_domain`` option instead!
+    ``sonata_type_translatable_choice`` is deprecated and will be removed in 4.0. Use form type ``choice`` with ``translation_domain`` option instead!
```

Comment:
IMHO, deprecated warning has nothing to do on the documentation.

We have the upgrade and code warning for that.